source/views/guides/getting-started.html.slim

```diff
@@ -61,13 +67,16 @@ main
           project's code directly, we don't add it to `Cargo.toml`. Instead, we
           just install it on our system.
 
+          The CLI has the same features as the diesel library and you should pass
+          the same as you passed in `Cargo.toml`.
+
           [diesel-cli]: https://github.com/sgrif/diesel/tree/master/diesel_cli
 
         .demo__example
           .demo__example-browser
             pre.demo__example-snippet
               code
-                | cargo install diesel_cli
+                | cargo install diesel_cli --no-default-features --features postgres
```

Comment:
Oh yes, I meant: Don't change that line at all, but supply the content of your change in prose form below :)